src/main/java/com/asana/Client.java

```diff
@@ -20,8 +21,7 @@
 /**
  * Main Asana client class
  */
-public class Client
-{
+public class Client {
```

Comment:
Lots of style changes to this file that seem unrelated, does this make it consistent with the rest of the repo?